src/main/kotlin/com/msg/gauth/domain/user/service/ChangePasswordService.kt

```diff
@@ -17,11 +18,15 @@ class ChangePasswordService(
 ){
     fun execute(passwordChangeReqDto: PasswordChangeReqDto){
         val currentUser = userUtil.fetchCurrentUser()
-        val emailAuth = emailAuthRepository.findById(currentUser.email)
-            .orElseThrow{ throw EmailNotVerifiedException() }
+        val emailAuth = emailAuthRepository.findByIdOrNull(currentUser.email)
+            ?: throw EmailNotVerifiedException()
+
         if(!emailAuth.authentication)
             throw EmailNotVerifiedException()
-        val update = currentUser.update(passwordEncoder.encode(passwordChangeReqDto.password))
+
+        val update = passwordChangeReqDto.toEntity(currentUser,
+            passwordEncoder.encode(passwordChangeReqDto.password))
+
```

Comment:
줄바꿈 위치가 한눈에 보기 어려운거같은데 이렇게나

```suggestion

        val update = passwordChangeReqDto.toEntity(
            currentUser,
            passwordEncoder.encode(passwordChangeReqDto.password)
        )

```

Reply:
혹은 이렇게 바꾸는거 어떤가요?
```suggestion

        val update = passwordChangeReqDto.toEntity(
            currentUser, passwordEncoder.encode(passwordChangeReqDto.password)
        )

```

Reply:
혹은 1번째에 파라미터 이름 명시도 해주는거도 좋을거같아요

Reply:
```kt
userRepository.save(
            passwordChangeReqDto.toEntity(
                currentUser, passwordEncoder.encode(passwordChangeReqDto.password)
            )
        )
``` 그냥 이렇게 하는건 어때요?

Reply:
엄.. 이부분은 스타일적인 부분인거 같아  @dolong2 @KimTaeO 랑 같이 협의하는게 좋을거같아요

제 개인 취향은 변수로 빼는 쪽이기는 합니다

Reply:
다른 곳에도 변수로 빼져 있어서 일단 변수로 빼놓고 처리하겠습니다.